--- wordpress-popular-posts.php	2012-12-21 07:52:20.000000000 -0800
+++ wordpress-popular-posts.php.bak	2012-09-13 00:18:33.000000000 -0700
@@ -162,7 +162,7 @@
 		function widget($args, $instance) {
 			//$args['widget_id'];
 			extract($args);
-			echo "<!-- Wordpress Popular Posts Plugin -->"."\n";
+			echo "<!-- Wordpress Popular Posts Plugin v". $this->version ." [W] [".$instance['range']."]". (($instance['markup']['custom_html']) ? ' [custom]' : ' [regular]') ." -->"."\n";
 			echo $before_widget . "\n";
 			
 			// has user set a title?
@@ -179,7 +179,7 @@
 			
 			echo $this->get_popular_posts($instance);			
 			echo $after_widget . "\n";
-			echo "<!-- End Wordpress Popular Posts Plugin -->"."\n";
+			echo "<!-- End Wordpress Popular Posts Plugin v". $this->version ." -->"."\n";
 		}
 
 		// updates each widget instance when user clicks the "save" button
@@ -1060,7 +1060,7 @@
 					}
 					// date
 					if ( $instance['stats_tag']['date']['active'] ) {						
-						$stats .= ($stats == "") ? "<span class=\"wpp-date\">" ." {$date}</span>" : " <span class=\"wpp-date\">" ." {$date}</span>";
+						$stats .= ($stats == "") ? "<span class=\"wpp-date\">" . __('posted on', 'wordpress-popular-posts')." {$date}</span>" : " | <span class=\"wpp-date\">" . __('posted on', 'wordpress-popular-posts') ." {$date}</span>";
 					}
 					
 					// POST THUMBNAIL
